jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -366,15 +423,18 @@ export default function AccountFieldsEditor(props: AccountFieldsEditorProps) {
       <div className={styles.row}>
         {/* Newsletter subscription opt-in */}
         {isFieldToBeDisplayed('newsletter_subscription') && (
-          <div className={styles.field}>
-            <Checkbox
-              checked={props.values.newsletter_subscription}
-              onChange={(isChecked: boolean) =>
-                onAnyFieldChange('newsletter_subscription', isChecked)
-              }
-              label={getLabel('newsletter_subscription')}
-            />
-          </div>
+          <>
+            <div className={styles.field}>
+              <label className={styles.checkboxLabel}>{t('Newsletter')}</label>
+              <Checkbox
+                checked={props.values.newsletter_subscription}
+                onChange={(isChecked: boolean) =>
+                  onAnyFieldChange('newsletter_subscription', isChecked)
+                }
+                label={getLabel('newsletter_subscription')}
+              />
+            </div>
+          </>
```

Comment:
The React fragment here `<></>` could probably get removed - there's only one child element.